### DISCUSSION TITLE
shadow outline issue in button group layout

### DISCUSSION THREAD

**Discussion by linusang** [tailwindlabs/tailwindcss#2569] [Help]
### Describe the problem:

the focus of a button in a button group using the focus:shadow-outline causes the outline to be blocked by "higher" elements

### Link to a minimal reproduction:

![image](https://user-images.githubusercontent.com/1967941/96151928-880eea80-0f3e-11eb-8e3e-c98fb7ba0476.png)

any thoughts on how to tackle this?

**Comment by adamwathan** [comment]
You'll want to adjust the z-index of the item when it's focused as currently the button on the right is positioned higher in the z-index tree.

You can enable focus variants for z-index in the config here:

https://tailwindcss.com/docs/z-index#responsive-and-pseudo-class-variants

Here's a Tailwind Play demonstrating what you want:

https://play.tailwindcss.com/mTt8vwZrdI
